docs/reference/composites.rst

```diff
@@ -258,3 +258,69 @@ Methods
    VirtualGraphComposite.sample
    VirtualGraphComposite.sample_ising
    VirtualGraphComposite.sample_qubo
+
+Reverse
+=======
+
+Composites that do batch reverse annealing. Can accept batch initial states or a set
+of schedules.
+
+BatchReverseComposite
+---------------------
+
+Accepts multiple initial states to initialize from.
+
+.. autoclass:: BatchReverseComposite
+
+
+Properties
+~~~~~~~~~~
+
+.. autosummary::
+   :toctree: generated/
+
+   BatchReverseComposite.child
+   BatchReverseComposite.children
+   BatchReverseComposite.properties
+   BatchReverseComposite.parameters
+
+
+Methods
+~~~~~~~
+
+.. autosummary::
+   :toctree: generated/
+
+   BatchReverseComposite.sample
+   BatchReverseComposite.sample_ising
+   BatchReverseComposite.sample_qubo
+
+ReverseAdvanceComposite
+-----------------------
+
+Advances an initial state using reverse annealing along a
```

Comment:
Not sure that "Advances" is clear here: "Reverse anneals from an initial state ..."

Reply:
reverse anneal from an initial state is different than what this composite is doing

Reply:
to me, from an initial state would mean taking the same state and creating submissions with the same initial state with different schedules. This composite chooses an initial state from the returned samples before the next submission with a new schedule

Reply:
Thank you, I went through the code again now and I see what you mean. I hadn't realized that `reinitialize_state` is playing two parts here: (1) for each RA submission it determines whether to do a "random walk" or a series of `num_reads` anneals from the same initial state (2) for your composite it determines whether to initialize with the last sample or most-found lowest-energy sample.

Because the docstring's current spacing problem will mess the format on ReadTheDocs ("schedules" in the 2nd line is under the "o" of Composite), not to mention the uncapitalized "i"s, I'll take another crack at it:

"""Composite that reverse anneals an initial sample through a sequence of anneal
     schedules. If you don not specify an initial sample, a random sample is used for the first
     submission. By default, each subsequent submission selects the most-found lowest-energy
     sample as its initial state. If you set `reinitialize_state` to False, which makes each submission 
     behave like a random walk, the subsequent submission selects the last returned sample as
     its initial state.

Reply:
i'm so good at writing... thanks a lot Joel!